static/app/views/dashboardsV2/widgetCard/transformMetricsResponseToSeries.tsx

```diff
@@ -0,0 +1,23 @@
+import {SessionApiResponse} from 'sentry/types';
```

Comment:
Should we rename this file to `transformSessionsResponseToSeries`? Same for the table.